aws_export_credentials/aws_export_credentials.py

```diff
@@ -194,7 +194,11 @@ def main():
             os.environ['AWS_DEFAULT_REGION'] = region_name
 
         command = ' '.join(shlex.quote(arg) for arg in args.exec)
-        sys.exit(os.system(command))
+        code = os.system(command)
```

Comment:
@benkehoe after doing some testing and driving myself nuts, I finally stumbled on this bit shifting that I needed to do...